src/components/Tags.tsx

```diff
@@ -1,17 +1,22 @@
+import { differenceBy, isEqual, uniq } from 'lodash';
 import * as React from 'react';
-import { MBTTAGS } from './TagSuggestions';
-
 //Dependencies for drag n drop tags with suggestions
 //npm install --save react-tag-input
 //npm install --save react-dnd@5.0.0
 //npm install --save react-dnd-html5-backend@3.0.2
 import { WithContext as ReactTags } from 'react-tag-input';
-import { stringify, v4 as uuid } from 'uuid';
+import { assertModelArrayType, Segment, SegmentTag } from 'types';
+import { MBTTAGS } from './TagSuggestions';
+
+type ReactTag = {
+  id: string;
+  text: string;
+};
 
-const suggestions = MBTTAGS.map(mbttags => {
+const suggestions: ReactTag[] = MBTTAGS.map(mbttag => {
   return {
-    id: uuid(),
```

Comment:
No need for uuid since the tag names are globally unique.

Reply:
They are typed by hand. There is a chance of duplicates I guess. Not a big issue anyway.

Reply:
They are typed by hand. There is a chance of duplicates I guess. Not a big issue anyway.